src/lib/geospatial.ts

```diff
@@ -0,0 +1,43 @@
+interface LatLng {
+  lat: number
+  lng: number
+}
+
+const EARTH_RADIUS_IN_METERS = 6371e3 // https://en.wikipedia.org/wiki/Earth_radius#Mean_radius
```

Comment:
You might be surprised at how [contentious](https://en.wikipedia.org/wiki/Talk:Great-circle_distance/Archive_1) this number is  😅 (_"All you have is a bunch of hand-waving arguments…"_)

Below are some util functions to support distance calculations.